erpnext/stock/doctype/delivery_trip/delivery_trip.py

```diff
@@ -33,6 +33,7 @@ def on_submit(self):
 		self.update_delivery_notes()
 
 	def on_update_after_submit(self):
+		self.is_visited()
```

Comment:
Rename this to what the function is supposed to do, something like `update_delivery_note_status`